catalog/app/components/Markdown/Markdown.js

```diff
@@ -189,7 +216,7 @@ export const Container = RT.composeComponent('Markdown.Container',
   }),
   withStyles(() => ({
     root: {
-      overflow: 'auto',
```

Comment:
why rules reordering here? i think it's a nice convention to put element's rules at the top with all the nested rules after that

Reply:
OK